nn-specials/src/main/java/org/frankframework/extensions/api/ApiStreamPipe.java

```diff
@@ -149,10 +148,10 @@ private String selectMessageKey(String slotId, String messageId) throws JdbcExce
 	}
 
 	private String selectMessage(String messageKey) throws JdbcException {
-		String query = "SELECT MESSAGE FROM IBISSTORE WHERE MESSAGEKEY='" + messageKey + "'";
+		String query = "SELECT MESSAGE FROM IBISSTORE WHERE MESSAGEKEY='?'";
```

Comment:
Jammer dat de table-name hier niet configureerbaar is, op veel plaatsen kan dat juist wel... Nou ja, NN probleempje dus zal voor hun niet nodig zijn.